ui/server/src/main/scala/pl/touk/nussknacker/ui/db/DatabaseInitializer.scala

```diff
@@ -24,35 +20,29 @@ object DatabaseServer {
   case class Config(dbFilePath:String, dbName:String, user:String, password:String, port:Option[Int], enabled: Option[Boolean])
 }
 
-class DatabaseInitializer(dbConfig: DbConfig) {
-  def initDatabase(): Unit = {
-    migrateIfNeeded(dbConfig)
-  }
-
-  private def migrateIfNeeded(dbConfig: DbConfig) = {
-    val flyway = Flyway.configure()
-      .dataSource(new DatabaseDataSource(dbConfig.db))
-      .baselineOnMigrate(true)
+object DatabaseInitializer {
 
-    val flywayWithDbProfile = dbConfig.driver match {
-      case HsqldbProfile => flyway.locations("db/migration/hsql", "db/migration/common")
-      case PostgresProfile => flyway.locations("db/migration/postgres", "db/migration/common")
-    }
-    flywayWithDbProfile.load().migrate()
+  def initDatabase(path: String, config: Config): Unit = {
```

Comment:
There should be only dbConfig.. Why You put path and config as separeted params?

Reply:
First of all the class named DbConfig is not a configuration class at all.It's a Slick database "handle". :)
Flyway uses a JDBC datasource to configure DB schema, but Slick doesn't expose the instance it uses internally. There was a tricky proxy here mimicking a java.sql.DataSource with a possible connection closing issue. 
Moreover there's no way to extract a db schema that Slick uses when connected to a database.
With this PR the FlyWay initializing method would have exactly the same signature as Slick initialization.

Reply:
I'd suggest renaming DbConfig class. It seems confusing now.

Reply:
You are right. So ;) Let's do some cleaning here.. If you would be so kind, please do
- rename DbConfig to DbHandler
- create case class for DbConfig with necessary fields

After thath we can use everywhere DbConfig.

Reply:
To be done in a separate PR.